test/numerical/TestLeakyRelu.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//====-- TestLeakyRelu.cpp - test GEMM code -======================================//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file contains the code to test LeakyRelu code.
+//
+//===----------------------------------------------------------------------===//
+
+// Common.hpp needs to be included first to correctly surpress the rapidcheck.h
+// warnings.
+#include "Common.hpp"
+
+#include "src/Runtime/OMTensorHelper.hpp"
+
+static const llvm::StringRef SHARED_LIB_BASE("./TestLeakyRelu_main_graph");
+
+using namespace mlir;
+
+namespace onnx_mlir {
+namespace test {
+
+void *omTensorGetAllocatedPtr(OMTensor *tensor);
+template <typename TYPE>
+void omPrintAsPython(OMTensor *tensor, std::string name) {
```

Comment:
First, the `om` prefix is reserved for onnx mlir interface. But in any case, I have added support in the ModelLib.hpp
```C++
  void printTensor(
      const std::string varName, const OMTensor *t, bool asNumpy = true) const;
```

where you can directly print it as a numpy var, which is great for Jupiter Notebook testing. So please reuse the existing code instead of having a new one, thanks.

Reply:
@AlexandreEichenberger Thank you for the comments.
I simply removed this code, because the code is not used, but is mistakenly copied from TestGemm.cpp .
I suppose that we can simply remove this code, and the numpy print function is supported by ModelLibBuilder.
Your comments are welcome.